### PR TITLE
chore: clarify email reply text

### PR DESCRIPTION
**Pull request by kleenkanteen**
For #4806

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -1673,8 +1673,8 @@ en:
         your_note_html: "%{commenter} has reactivated one of your map notes near %{place}."
         commented_note: "%{commenter} has reactivated a map note you have commented on. The note is near %{place}."
         commented_note_html: "%{commenter} has reactivated a map note you have commented on. The note is near %{place}."
-      details: "More details about the note can be found at %{url}."
-      details_html: "More details about the note can be found at %{url}."
+      details: "Reply or learn more about the changeset at %{url}."
```
Comment (gravitystorm) [inline config/locales/en.yml:1676]: This looks like a copy+paste error - you’ve changed note to change set here (twice)

**Review by gravitystorm — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 1 above)
Comment (kleenkanteen) [reply config/locales/en.yml]: woops, fixed

**Comment by tomhughes** [comment]
That looks good to me now, thanks.

**Comment by kleenkanteen** [comment]
@tomhughes by the way, shouldn't PR's by default be squashed when merged? there's 2 commits on the master branch when there should be just one imo for a cleaner history:

image

There's a github setting that allows only squash merging:

image

Also why is there no development branch? That way, each commit to the master branch would be major changes, while each commit to to the development branch would be minor changes like my PR here.

**Comment by tomhughes** [comment]
No, you should ideally have squashed them together as explained at https://github.com/openstreetmap/openstreetmap-website/blob/master/CONTRIBUTING.md#pull-requests.

I didn't notice you hadn't done that or I would have asked you do so. I don't actually merge on github so the github settings are not relevant.

**Comment by gravitystorm** [comment]
> Also any thoughts on my developement branch question in my above post?

We merge all accepted PRs to master immediately, so that they can be immediately deployed, and the community immediately benefits from them. It doesn't matter whether the change is big or small, there's no reason to delay merging into master or to create extra complexity by having multiple long-running branches.

**Pull request merged**